src/issue-strategies/bug-maintenance.js

```diff
@@ -1,7 +1,18 @@
+import * as jiraOperations from '../jira-operations.js';
+
 export function apply(issue, jiraClientAPI) {
   if(issue === null || issue.fields.status.statusCategory.colorName !== 'yellow') {
     return Promise.reject(new Error(`Issue ${issue.key} is not open to commit against`));
   }
 
+  if(issue.fields.issuelinks){
+    return jiraOperations.findParent(issue, jiraClientAPI).then((parent)=>{
```

Comment:
`parent` doesn't need to be wrapped since it's a single arg. Spaces around the arrow are also nice.